From akpm@linux-foundation.org Fri Mar 14 15:38:01 2008
Return-Path: <akpm@linux-foundation.org>
Received: from imap.linux.ibm.com ([unix socket]) by imap.linux.ibm.com
	(Cyrus v2.3.7-Invoca-RPM-2.3.7-7) with LMTPA; Fri, 14 Mar 2008 15:38:01
	-0400
X-Sieve: CMU Sieve 2.3
Received: by imap.linux.ibm.com (Postfix, from userid 101) id 9F96D1910234;
	Fri, 14 Mar 2008 15:38:01 -0400 (EDT)
X-Spam-TestScore:
	NO_REAL_NAME=0.55,SPF_NEUTRAL=1.379,TW_JB=0.077,TW_SR=0.077, TW_TK=0.077
X-Spam-TokenSummary: Bayes not run.
X-Spam-Checker-Version: SpamAssassin 3.1.7 (2006-10-05) on
	imap.linux.ibm.com
X-Spam-Level: **
X-Spam-Status: No, score=2.2 required=5.0 tests=NO_REAL_NAME,SPF_NEUTRAL,
	TW_JB,TW_SR,TW_TK autolearn=disabled version=3.1.7
X-Spam-Relay-Country: US ** US US US ** US US US US **
Received: from smtp.linux.ibm.com (smtp.linux.ibm.com [9.26.4.197]) by
	imap.linux.ibm.com (Postfix) with ESMTP id 2FD92191015F for
	<cmm@imap.linux.ibm.com>; Fri, 14 Mar 2008 15:37:55 -0400 (EDT)
Received: from localhost (localhost.localdomain [127.0.0.1]) by
	smtp.linux.ibm.com (Postfix) with ESMTP id 04A9EC0F0 for
	<cmm@linux.ibm.com>; Fri, 14 Mar 2008 15:37:55 -0400 (EDT)
X-Virus-Scanned: amavisd-new at linux.ibm.com
Received: from VMSDVMA.POK.IBM.COM (vmsdvma.pok.ibm.com [9.56.231.65]) by
	smtp.linux.ibm.com (Postfix) with ESMTP id C408AC078 for
	<cmm@linux.ibm.com>; Fri, 14 Mar 2008 15:37:54 -0400 (EDT)
Received:  by VMSDVMA.POK.IBM.COM (IBM VM SMTP Level 520) via spool with
	SMTP id 2710 ; Fri, 14 Mar 2008 15:36:32 EDT
Received: by vmsdvma.vnet.ibm.com (xagent2 5.9.6) via xagsmtp3 with spool
	id 6363 for cmm@linux.vnet.ibm.com; Fri, 14 Mar 2008 15:36:32 -0400 (EDT)
Received: from d01relay07.pok.ibm.com [9.56.227.147] by VMSDVMA.POK.IBM.COM
	(IBM VM SMTP Level 520) via TCP with ESMTP ; Fri, 14 Mar 2008 15:36:32 EDT
Received: from d01av05.pok.ibm.com (d01av05.pok.ibm.com [9.56.224.195]) by
	d01relay07.pok.ibm.com (8.13.8/8.13.8/NCO v8.7) with ESMTP id
	m2EJbrnn1798244 for <cmm@us.ibm.com>; Fri, 14 Mar 2008 15:37:53 -0400
Received: from d01av05.pok.ibm.com (loopback [127.0.0.1]) by
	d01av05.pok.ibm.com (8.12.11.20060308/8.13.3) with ESMTP id m2EJZWId025943
	for <cmm@us.ibm.com>; Fri, 14 Mar 2008 15:35:32 -0400
Received: from d01as02.pok.ibm.com (d01as02 [9.56.224.38]) by
	d01av05.pok.ibm.com (8.12.11.20060308/8.12.11) with ESMTP id m2EJZWLY025939
	(version=TLSv1/SSLv3 cipher=DHE-RSA-AES256-SHA bits=256 verify=OK) for
	<cmm@us.ibm.com>; Fri, 14 Mar 2008 15:35:32 -0400
Received: from e6.ny.us.ibm.com (e6.pok.ibm.com [9.56.232.146]) by
	d01as02.pok.ibm.com (8.12.11.20060308/8.12.11) with ESMTP id m2EJbpPl009242
	(version=TLSv1/SSLv3 cipher=DHE-RSA-AES256-SHA bits=256 verify=OK) for
	<cmm@us.ibm.com>; Fri, 14 Mar 2008 15:37:51 -0400
Received: from smtp1.linux-foundation.org (smtp1.linux-foundation.org
	[140.211.169.13]) by e6.ny.us.ibm.com (8.13.8/8.13.8) with ESMTP id
	m2EJdm1B015508 (version=TLSv1/SSLv3 cipher=DHE-RSA-AES256-SHA bits=256
	verify=FAIL) for <cmm@us.ibm.com>; Fri, 14 Mar 2008 15:39:49 -0400
Received: from imap1.linux-foundation.org (imap1.linux-foundation.org
	[140.211.169.55]) by smtp1.linux-foundation.org
	(8.14.2/8.13.5/Debian-3ubuntu1.1) with ESMTP id m2EJbY88012961
	(version=TLSv1/SSLv3 cipher=DHE-RSA-AES256-SHA bits=256 verify=NO); Fri, 14
	Mar 2008 12:37:36 -0700
Received: from localhost.localdomain (localhost [127.0.0.1]) by
	imap1.linux-foundation.org (8.13.5.20060308/8.13.5/Debian-3ubuntu1.1) with
	ESMTP id m2EJac7T015842; Fri, 14 Mar 2008 12:36:38 -0700
Message-Id: <200803141936.m2EJac7T015842@imap1.linux-foundation.org>
Subject: + jbd2-fix-kernel-doc-notation.patch added to -mm tree
To: mm-commits@vger.kernel.org
Cc: randy.dunlap@oracle.com, cmm@us.ibm.com
From: akpm@linux-foundation.org
Date: Fri, 14 Mar 2008 12:36:38 -0700
X-MIMEDefang-Filter: lf$Revision: 1.188 $
X-Scanned-By: MIMEDefang 2.63 on 140.211.169.13
X-Xagent-From: akpm@linux-foundation.org
X-Xagent-To: cmm@linux.vnet.ibm.com
X-Xagent-Gateway: vmsdvma.vnet.ibm.com (XAGENTU at VMSDVMA)
X-Evolution-Source: imap://cmm@imap.linux.ibm.com/
Content-Transfer-Encoding: 8bit
Mime-Version: 1.0


The patch titled
     jbd2: fix kernel-doc notation
has been added to the -mm tree.  Its filename is
     jbd2-fix-kernel-doc-notation.patch

Before you just go and hit "reply", please:
   a) Consider who else should be cc'ed
   b) Prefer to cc a suitable mailing list as well
   c) Ideally: find the original patch on the mailing list and do a
      reply-to-all to that, adding suitable additional cc's

---
 fs/jbd2/journal.c     |    5 +++--
 fs/jbd2/transaction.c |    3 ++-
 2 files changed, 5 insertions(+), 3 deletions(-)

Index: linux-2.6.25-rc6/fs/jbd2/journal.c
===================================================================
--- linux-2.6.25-rc6.orig/fs/jbd2/journal.c	2008-03-24 14:36:48.000000000 -0700
+++ linux-2.6.25-rc6/fs/jbd2/journal.c	2008-03-24 14:37:26.000000000 -0700
@@ -1007,13 +1007,14 @@ fail:
  */
 
 /**
- *  journal_t * jbd2_journal_init_dev() - creates an initialises a journal structure
+ *  journal_t * jbd2_journal_init_dev() - creates and initialises a journal structure
  *  @bdev: Block device on which to create the journal
  *  @fs_dev: Device which hold journalled filesystem for this journal.
  *  @start: Block nr Start of journal.
  *  @len:  Length of the journal in blocks.
  *  @blocksize: blocksize of journalling device
- *  @returns: a newly created journal_t *
+ *
+ *  Returns: a newly created journal_t *
  *
  *  jbd2_journal_init_dev creates a journal which maps a fixed contiguous
  *  range of blocks on an arbitrary block device.
Index: linux-2.6.25-rc6/fs/jbd2/transaction.c
===================================================================
--- linux-2.6.25-rc6.orig/fs/jbd2/transaction.c	2008-03-24 14:33:12.000000000 -0700
+++ linux-2.6.25-rc6/fs/jbd2/transaction.c	2008-03-24 14:37:26.000000000 -0700
@@ -1462,7 +1462,8 @@ int jbd2_journal_stop(handle_t *handle)
 	return err;
 }
 
-/**int jbd2_journal_force_commit() - force any uncommitted transactions
+/**
+ * int jbd2_journal_force_commit() - force any uncommitted transactions
  * @journal: journal to force
  *
  * For synchronous operations: force any uncommitted transactions
